src/core/api_logs.tf

```diff
@@ -52,3 +66,36 @@ resource "azurerm_api_management_api_diagnostic" "apim_logs" {
     body_bytes = 8192
   }
 }
+
+# api info log without request/response body
+resource "azurerm_api_management_api_diagnostic" "apim_info_logs" {
+  for_each = toset(local.api_info_log)
+
+  identifier               = "applicationinsights"
+  resource_group_name      = azurerm_resource_group.rg_api.name
+  api_management_name      = module.apim.name
+  api_name                 = each.key
+  api_management_logger_id = module.apim.logger_id
+
+  sampling_percentage       = 100
+  always_log_errors         = true
+  log_client_ip             = true
+  verbosity                 = "information"
+  http_correlation_protocol = "W3C"
+
+  frontend_request {
+    body_bytes = 0
+  }
+
+  frontend_response {
+    body_bytes = 0
+  }
+
+  backend_request {
+    body_bytes = 0
+  }
+
+  backend_response {
+    body_bytes = 0
+  }
```

Comment:
```suggestion
```
according to https://registry.terraform.io/providers/hashicorp/azurerm/2.99.0/docs/resources/api_management_api_diagnostic#backend_request

frontend_request, frontend_response, backend_request, backend_response are optional so maybe better remove them.

Anyway setting to 0 body request are not saved into logs.

Since PM is critical application a test in DEV/UAT can be useful